python/GafferUI/VectorDataWidget.py

```diff
@@ -811,20 +818,26 @@ def __init__( self, data, parent=None, editable=True, header=None, columnToolTip
 		self.__data = data
 		self.__editable = editable
 		self.__header = header
-		self.__columnToolTips = columnToolTips
+		self.__columnToolTips = []
 		self.__columnEditability = columnEditability
 
 		self.__columns = []
 		self.__accessors = []
-		for d in self.__data :
-			accessor = _DataAccessor.create( d )
+		index = 0
+		for name, d in self.__data.items() :
+			accessor = _DataAccessor.create( d, name = name )
 			assert( accessor is not None )
 			for i in range( 0, accessor.numColumns() ) :
-				self.__columns.append( IECore.Struct( accessor=accessor, relativeColumnIndex=i ) )
+				self.__columns.append( IECore.Struct( accessor=accessor, relativeColumnIndex=i, index = index ) )
```

Comment:
`index` is the original column index as far as `data` is concerned, right? We have some jiggery pokery for going backwards and forwards on this in `VectorDataWidget.columnToDataIndex()` and `VectorDataWidget.dataToColumnIndex()`. I wonder if we should avoid the redundancy one way or another?